plexpy/notification_handler.py

```diff
@@ -516,11 +563,35 @@ def build_notify_text(session=None, timeline=None, state=None):
                         'summary': metadata['summary'],
                         'tagline': metadata['tagline'],
                         'rating': metadata['rating'],
-                        'duration': duration
+                        'duration': duration,
+                        'action': state,
+                        'streams': stream_count
                         }
 
     # Default subject text
     subject_text = 'PlexPy (%s)' % server_name
+    # Default scripts args
+    script_args = ''
```

Comment:
It will quote if the user enters `{param}` but it won't quote if the user enters `"{param}"`.